test/integration/node/lifecycle_test.go

```diff
@@ -297,168 +292,89 @@ func TestTaintBasedEvictions(t *testing.T) {
 			// Waiting for all controllers to sync
 			externalInformers.Start(testCtx.Ctx.Done())
 			externalInformers.WaitForCacheSync(testCtx.Ctx.Done())
-			testutils.SyncInformerFactory(testCtx)
```

Comment:
are you sure this line is not necessary?

Reply:
Yes, actually we cannot even call it in the current setup. because the function calls `Start` and `WaitForCacheSync` on `testCtx.InformerFactory` which is nil in this test (after getting rid of scheduler). 
However, using this function could replace the two lines above `externalInformers.Start(testCtx.Ctx.Done())` and `externalInformers.WaitForCacheSync(testCtx.Ctx.Done())`  if we earlier assigned `testCtx.InformerFactory = externalInformers`, but it isn't a clear gain to me. The function `SyncInformerFactory` seems to only be used by tests for Scheduler currently.